code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -114,6 +114,8 @@ var/global/list/datum/stack_recipe/wood_recipes = list ( \
 	new/datum/stack_recipe("dog bed", /obj/structure/bed/dogbed, 10, time = 10, one_per_turf = 1, on_floor = 1), \
 	new/datum/stack_recipe("display case chassis", /obj/structure/displaycase_chassis, 5, one_per_turf = 1, on_floor = 1), \
 	new/datum/stack_recipe("wooden buckler", /obj/item/weapon/shield/riot/buckler, 20, time = 40), \
+	new/datum/stack_recipe("apiary", /obj/structure/beebox, 40, time = 50),\
```

Comment:
with a /fifty stack of wood (the bought type) you get an apiary and 2 frames (that's 2/3rds of the max capacity), and towercaps give you 25% of their potency (+1) in planks, potency is 50 by default for towercaps
so that's 12.5 planks (so 13, it rounds up) per log.

so in like, 4 towercaps you could build an apiary, with 2 towercaps you could get 3 frames, so from 6 towercaps you have a whole apiary setup.